chart/templates/esp-deployment.yaml

```diff
@@ -70,14 +70,11 @@ spec:
 {{- if .Values.stubConnector.enabled }}
         - name: CONNECTOR_NODE_METADATA_TRUSTSTORE
           valueFrom:
-            secretKeyRef:
-              name: {{ .Release.Name }}-connector-metadata
-              key: metadataSigningTruststoreBase64
+            configMapKeyRef:
```

Comment:
Mainly so that we can see exactly how the truststore, plus keys and certs were created.
The fact that the configmap resolves to files in the `/pki` directory when volume mounted makes it even more attractive.
Granted this would be obsolete if/when we move to PaaS.